plugins/mac-virtualcam/src/obs-plugin/Defines.h

```diff
@@ -0,0 +1,24 @@
+//
+//  Defines.h
+//  obs-mac-virtualcam
+//
+//  Created by John Boiles  on 5/27/20.
+//
+//  obs-mac-virtualcam is free software: you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License as published by
+//  the Free Software Foundation, either version 2 of the License, or
+//  (at your option) any later version.
+//
+//  obs-mac-virtualcam is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+//  GNU General Public License for more details.
+//
+//  You should have received a copy of the GNU General Public License
+//  along with obs-mac-virtualcam. If not, see <http://www.gnu.org/licenses/>.
+
+#define PLUGIN_NAME "mac-virtualcam"
+#define PLUGIN_VERSION "0.0.1"
```

Comment:
I'm changing this to 1.3.0 in my branch to help us debug/understand any sort of issues between folks running the legacy version of my plugin (<=1.2.0) vs folks running the new official OBS version